apps/protocol-frontend/src/hooks/useAttestationBulkMint.ts

```diff
@@ -49,7 +49,7 @@ const useAttestationBulkMint = () => {
 
         if (results.length > 0) {
           toast.success({
-            title: 'Attestation successfully minted',
+            title: 'Attestation Successfully Minted',
```

Comment:
Does successfully and minted need to be capitalized

Reply:
Yup, the title should be capitalized. Will fix